css/style.css

```diff
@@ -15,5 +14,195 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+h1 {
+  margin: auto;
+  margin-top: auto;
+  font-size: 40px;
+}
+h2 {
+  margin: auto;
+  margin-top: auto;
+  font-size: 20px;
+}
+h3 {
+  text-align: center;
+  font-size: 24px;
+  top: 30%;
+}
+
+#navbar-button {
+  margin-left: 25px;
+  margin-right: 25px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction: row` is the default so no need to add (tho it doesn't do any harm either!)